migrations/server/mysql/0003_add_sha256_tuf_files.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `tuf_files` ADD COLUMN `sha256` CHAR(64) DEFAULT NULL, ADD INDEX `sha256` (`sha256`);
+
+UPDATE `tuf_files` SET `sha256` = SHA2(`data`, 256);
```

Comment:
nit: might be helpful to link to the [docs for SHA2](https://dev.mysql.com/doc/refman/5.5/en/encryption-functions.html#function_sha2), so it's clear that it's compatible with the golang sha256 -- I have less experience with mysql so I had to look it up, but it might be clearer for others.